js/apps/admin-ui/src/groups/GroupsSection.tsx

```diff
@@ -53,6 +56,9 @@ export default function GroupsSection() {
   const location = useLocation();
   const id = getLastId(location.pathname);
 
+  const [searchParams] = useSearchParams();
+  const lazy = searchParams.get("lazy") || "false";
```

Comment:
Shouldn't this be a boolean instead of a String?

Something like, 
```suggestion
  const lazy = searchParams.has("lazy") || false;
```

Reply:
right if we use `has` the value doesn't matter so it will be `?lazy` then it will hide the tree view initially and it will only show the root and search